### DISCUSSION TITLE
Improving Dutch translations for Alf.io

### DISCUSSION THREAD

**Discussion by StanMar-bit** [alfio-event/alf.io#1235] [translations]
Currently I'm using the alf.io software for the ticket sale of a charity fundraiser. So far we're really pleased and impressed with the software :)
But we noticed som inconsistencies within the Dutch translations, for example in the invoicing data labels. "Adres" is double, it should be "factuuradres" =billing address, and "adres"=address. We also noticed several sort like inconsistencies in in the emails.

That's way I joined the POEditor, to improve the Dutch translations, but I'm still waiting for approval.
I'm born and raised in the Netherlands and therefore a native speaker :) So I hope you will aprove me and allow me to improve and add translations.
My profile name is Stan on POEditor

**Comment by cbellone** [comment]
Hi! And thanks for helping us! 
You should be now allowed to modify texts. 

Once you're done with the modifications, please reply to this message and we'll integrate them

**Comment by StanMar-bit** [comment]
Hi, thanks for your fast reponse. I read all the translations and changed them where necessary. For now I'm done with the modifcations :)
If I notice a wrong translation in the future, I will change it on POEditor and let you know of the modification.
